octokit/result.go

```diff
@@ -1,9 +1,18 @@
 package octokit
 
 import (
+	"strconv"
+	"strings"
+
 	"github.com/jingweno/go-sawyer/mediaheader"
 )
 
+const (
+	oauthScopes         = "X-OAuth-Scopes"
+	oauthAcceptedScopes = "X-OAuth-Accepted-Scopes"
+	rateLimitRemaining  = "X-RateLimit-Remaining"
```

Comment:
There's also `X-RateLimit-Reset: 1428627160`, which is a Unix epoch. Worth exposing as a timeish?